idd/Energy+.idd.in

```diff
@@ -23681,16 +23706,21 @@ ZoneInfiltration:EffectiveLeakageArea,
 
 ZoneInfiltration:FlowCoefficient,
        \min-fields 8
-       \memo  Infiltration is specified as flow coefficient, schedule fraction, stack and wind coefficients, and
-       \memo  is a function of temperature difference and wind speed:
-       \memo  Infiltration=FSchedule * SQRT( (c * Cs*|(Tzone-Todb)|**n)**2 + (c* Cw*(s * WindSpd)**2n)**2 )
+       \memo Infiltration is specified as flow coefficient, schedule fraction, stack and wind coefficients, and
+       \memo is a function of temperature difference and wind speed:
+       \memo Infiltration=FSchedule * SQRT( (c * Cs*|(Tzone-Todb)|**n)**2 + (c* Cw*(s * WindSpd)**2n)**2 )
+       \memo If a Zone comprised of more than one Space is specified
+       \memo then this definition applies to all applicable spaces, and each instance will
+       \memo be named with the Space Name plus this Object Name.
```

Comment:
Here is an example where the use of these objects in zones and spaces is confusing. It says here if a ZoneInfiltration:FlowCoefficient specified for a zone will be applied to each space. For a 2 space zone, that would be 3 objects when in fact the intention is to only model the 2 objects at the space level. It seems it is the space level object that gets simulated and the zone level object is used as a master object. As the example file 5ZoneAirCooledWithSpaces shows, it's the spaces that are modeled when there are multiple space. Each ZoneInfiltration:DesignSpecification object is specified at the zone level and zones 3 and 5 have multiple spaces. Here Zone 3 and Zone 5 infiltration is not reported and the spaces are reported.

If I read this again I see that when a zone level object is specified and there is more than 1 space then the zone level object is applied (it doesn't say "also applies", it says "applies") to the spaces. So maybe the descriptions for these objects don't need to be revised.

![image](https://user-images.githubusercontent.com/5778156/182669541-1e5b5f6c-8676-44b6-a034-4e8bb21a897d.png)